src/main/java/com/codeborne/selenide/commands/SetValue.java

```diff
@@ -50,19 +50,13 @@ private void setValueForTextInput(Driver driver, WebElement element, String text
     else if (driver.config().fastSetValue()) {
       String error = setValueByJs(driver, element, text);
       if (error != null) throw new InvalidStateException(driver, error);
```

Comment:
I guess we cannot delete this part. When we set value by JavaScript, we still need to trigger `change` and other events. It can only be skipped when we set value by `sendKeys()` (because browser does it automatically anyway).

Reply:
But in this case based on setValueChangeEvent config we just add an extra "change" event.

In other case we trigger all other "keydown", "keypress", "input", "keyup", and now we also trigger them.